documentation/testing.md

```diff
@@ -804,7 +804,7 @@ testpmd> show port stats all
 
 # if checking job-trex-app logs, at the end, you should see that the execution was correct
 
-$ oc logs -n example-cnf job-trex-app-4jkn5
+$ oc logs -n example-cnf `oc get pod -n example-cnf|awk '{print $1}'|grep job-trex-app`
```

Comment:
same here, I believe another way can work here as described in my other comments.